lib/ansible/modules/windows/win_description.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, RusoSova
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: win_description
```

Comment:
I think `win_description` is too generic for what this module does but I am struggling to come up with an alternative. My best suggestion would be `win_system_description` which at least tells us that we are managing the system wide info. I think `win_system_info` is best but unfortunately the `_info` suffix is meant for modules that exclusively return information like `_facts` but not as an ansible fact.

Reply:
Instead of creating a new module from scratch, what do you think to integrate this feature into win_hostname and/or win_domain_membership modules?

Reply:
I personally don't think so, win_hostname is purely about the hostname and I don't think we should add more entries. The `win_domain_membership` is also solely about domain/workgroup membership and I don't see a reason for adding more onto that. I don't have a problem with having a module for this work, my main issue is that I think `win_description` is just a bit too generic.

Reply:
It makes sense to me to add win_description features into win_hostname module.

After all, on Windows you can set the computer description in the same window where you set the hostname and domain membership(under Control Panel\System and Security\System)

Reply:
By that logic `win_domain_membership` should also come under `win_hostname`. There's already an existing [hostname](https://docs.ansible.com/ansible/latest/modules/hostname_module.html) module for the Linux side and I prefer to keep the Windows variant as close to it as possible.

We can always link to this module in the [win_hostname](https://docs.ansible.com/ansible/latest/modules/win_hostname_module.html) docs using the seealso section

Reply:
I also agree with jborean93, a module should take over as possible only one task.